sdk/storage/azure_storage_blob/src/clients/blob_container_client.rs

```diff
@@ -79,18 +78,18 @@ impl BlobContainerClient {
     ///
     /// * `blob_name` - The name of the blob.
     /// * `options` - Optional configuration for the client.
-    pub fn blob_client(
-        &self,
-        blob_name: String,
-        options: Option<BlobClientOptions>,
-    ) -> Result<BlobClient> {
-        BlobClient::new(
-            self.endpoint().as_str(),
-            self.container_name().to_string(),
-            blob_name,
-            self.credential.clone(),
-            options,
-        )
+    pub fn blob_client(&self, blob_name: String) -> BlobClient {
+        let generated_blob_client = GeneratedBlobClient {
```

Comment:
Instead of building a new generated sub-client can we use `self.client.get_blob_client`?

Reply:
Good callout-- fixed this here and other instance in BSC